components/gallery/List.vue

```diff
@@ -0,0 +1,303 @@
+
+        },
+        preparedList() {
+            let tempData = this.items
+            let {right, left} = this.shapeListData(tempData, 3)
+            let trimmed = this.shapeListData(left, 2)
+
+            return [
+                [
+                    {
+                        listItems: trimmed.left
+                    },
+                ],
+                [
+                    {
+                        listItems: trimmed.right
+                    },
+                    {
+                        listItems: right
+                    }
+                ]
+
+            ]
+        },
+    },
+    methods: {
+        videoLoaded() {
+            this.isVideoLoaded = true
+        },
+        imageLoaded() {
+            this.isImageLoaded = true
+        },
+        videoError() {
+            this.isVideoError = true
+        },
+        imageError() {
+            this.isImageError = true
+        },
+
+        mouseOver(item) {
+            this.caption = item.featuredImage.node.caption !== null ? item.featuredImage.node.caption : `Who the fuxt is ${item.title}?`
+            this.image = item.featuredImage.node
+        },
+        mouseOut() {
+            this.isVideoLoaded = false
+            this.isImageLoaded = false
+            this.isVideoError = false
+            this.isImageError = false
+            this.caption = ''
+            this.image = null
+        },
+        shapeListData(array, offset) {
+            let count = array.length
+            let trimLastCount = Math.floor(count / offset)
+            let rightArray = _takeRight(array, trimLastCount)
+            let leftArray = _dropRight(array, trimLastCount)
+
+            return {
+                left: leftArray,
+                right: rightArray
+            }
+        },
+    }
+
+}
+</script>
+
+
+<style lang="scss" scoped>
+// import font at the component level because it doesn't appear to be loaded globally
+@font-face {
+    font-family: 'RM Neue';
+    src: url('../../assets/fonts/RMNeue-Regular.woff2') format('woff2'),
+    url('../../assets/fonts/RMNeue-Regular.woff') format('woff');
+    font-weight: normal;
+    font-style: normal;
+}
+
+.gallery-list {
+    position: relative;
+    z-index: 0;
+
+    max-width: 1280px;
+    min-height: var(--unit-100vh);
+    margin: 0 auto;
+    padding: 0 var(--unit-gap);
+    box-sizing: border-box;
+
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
+    font-family: var(--font-primary);
+
+    .list {
+        margin: 0;
+        padding: 0;
+        list-style: none;
+    }
+
+    .list-main {
+        position: relative;
+        z-index: 20;
+
+        max-width: 1180px;
+        width: 100%;
+        height: 100%;
+        box-sizing: border-box;
+        margin: 0 auto;
+
+        display: flex;
+        justify-content: space-between;
+    }
+
+    .item-main {
+        flex: 1 1 37%;
+
+        &:last-child {
+            flex: 1 1 63%;
```

Comment:
These look like magic numbers, I'd say generally avoid this approach.

Reply:
Updated, please see the screencast which explains this madness.  
https://www.awesomescreenshot.com/video/6010161?key=211dc6030e4afd8d04052b7d26a6206f